cmd/bootstrap/template.go

```diff
@@ -11,7 +11,10 @@ env:
     ALLOW_OVERWRITE: true
     DISABLE_API: false`
 
-	operatorValuesYAML string = `Installation:
+	operatorValuesYAML string = `operatorkit:
+  resyncPeriod: "20s"
```

Comment:
The finalizer TTL in operatorkit is 3 * resync period. So its now 1 minute instead of 15 mins.

Each test run with app-build-suite takes at least 1 minute so this TTL should be short enough.